build/run

```diff
@@ -65,7 +65,7 @@ if [ "`uname -s`" != "Linux" ]; then
     #    /.netrc (bind mounted to host ${HOME}/.netrc)
     #    /.aws (bind mounted to host ${HOME}/.aws)
     #    /.docker (bind mounted to host ${HOME}/.docker)
-    #    /.download (bind mounted to host ${DOWNLOADDIR} if set)
+    #    /.download (bind mounted to host ${CACHE_DIR} if set)
```

Comment:
the .gitignore was changed from .download to .cache.  i assume this comment should be updated too?